mmds/src/lib.rs

```diff
@@ -16,15 +21,15 @@ use data_store::{Error as MmdsError, Mmds};
 use micro_http::{Body, Request, RequestError, Response, StatusCode, Version};
 
 lazy_static! {
-    // A static reference to a global Mmds instance. We currently use this for ease of access during
-    // prototyping. We'll consider something like passing Arc<Mutex<Mmds>> references to the
-    // appropriate threads in the future.
+    /// A static reference to a global Mmds instance. We currently use this for ease of access during
```

Comment:
nit: only the first phrase is the documentation, the next lines are more code comments and probably should not be part of the public documentation. I suggest you only keep the first line as documentation and for the rest you can leave them as they were before.